src/rendezvous/cancellationReason.ts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This file is improperly named, I think. Should be called `RendezvousFailureReason.ts`